File: agino_mobile/lib/features/farm/domain/entities/farm.dart
import 'package:equatable/equatable.dart';

// ignore: must_be_immutable
class Farm extends Equatable {
  int? farmId;
  final String name;
  final String postCode;
  final String city;
  int? userId;
  final String country;

  Farm(
      {this.farmId,
      required this.name,
      required this.postCode,
      required this.city,
      this.userId,
      required this.country});

  @override
  List<Object?> get props => [farmId, name, postCode, city, country];
}

File: AgriHelper.Api/AgriHelper.Core/Models/Farm.cs

namespace AgriHelper.Core.Models
{
    public class Farm
    {

        public int FarmId { get; set; }
        public string Name { get; set;}
        public string PostCode { get; set; }
        public string City { get; set; }
        public string Country { get; set; } 

        public ICollection<FarmUser>? FarmUsers { get; set; }

        public ICollection<Field> Fields { get; set; }

    }
}
